ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx

```diff
@@ -605,3 +606,24 @@ export function createExtraneousEdges(selectedNodeId: string): {
     };
     return { extraneousEgressEdge, extraneousIngressEdge };
 }
+
+// This function returns the ids of nodes that are connected to the selected node
+export function getConnectedNodeIds(
+    nodes: CustomNodeModel[],
```

Comment:
We don't seem to be using the `nodes` parameter.

Is it an artifact from an earlier version of the code that we could remove?

Reply:
Oh, yea probably. Good caatch